code/src/test/pcgen/core/CampaignTest.java

```diff
@@ -38,7 +38,6 @@
 /**
  * <code>CampaignTest</code> is ...
  * 
- * Last Editor: $Author$ Last Edited: $Date: 2008-11-11 18:34:19 -0500
  * (Tue, 11 Nov 2008) $
```

Comment:
can I do those as a separate PR? I'd prefer to keep this one entirely automated.